### PR TITLE
8 project issue reference

### PR DESCRIPTION
**Pull request by bealdav**
This module allows to end users to create an issue from any configured models
and link automatically the intial object to the issue with a reference field.

### CODE REVIEW

File: project_issue_reference/README.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Project Issue Reference
+=======================
+
+This module allows to end users to create an issue from any configured models
+and link automatically the intial object to the issue with a reference field.
```
Comment (pedrobaeza) [inline project_issue_reference/README.rst]: s/intial/initial

**Comment by rafaelbn** [comment]
Happy to test it when available in runbot

**Comment by pedrobaeza** [comment]
I have enabled runbot. @bealdav, please rebase the branch to get a new runbot build.

**Comment by bealdav** [comment]
thanks @pedrobaeza, needs yet some work from my side.

File: project_issue_reference/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Project Issue Reference',
+    'version': '0.2',
```
Comment (dreispt) [inline project_issue_reference/__openerp__.py:23]: 8.0.0.2.0

File: project_issue_reference/action.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
```
Comment (dreispt) [inline project_issue_reference/action.py:1]: I think the `-*-` are required

Comment (bealdav) [inline project_issue_reference/action.py:1]: see https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms
from here https://github.com/OCA/maintainer-tools/pull/124

**Comment by pedrobaeza** [comment]
Please rebase to get green tests

**Comment by bealdav** [comment]
Now replaced by https://github.com/OCA/project/pull/137/commits

**Pull request closed without merging**